sdk/containerregistry/azure-containerregistry/tests/test_anon_access.py

```diff
@@ -139,6 +169,9 @@ def test_update_repository_properties(self, containerregistry_anonregistry_endpo
 
     @acr_preparer()
     def test_update_tag_properties(self, containerregistry_anonregistry_endpoint):
+        if not self.is_public_endpoint(containerregistry_anonregistry_endpoint):
```

Comment:
I think you can do a `pytest.mark.skipif` instead, even though you're looking at a `self` parameter

Reply:
I get a `NameError: name 'self' is not defined` if I do this, also the parameter is passed in by the preparer so it wouldn't be available either.